back-end/controllers/fetchFriendUploads.js

```diff
@@ -9,16 +9,22 @@ Image.belongsTo(User, {foreignKey: 'uploaded_image_user_id', as: 'UploadedImageU
 
 
 const fetchFriendUploads = (req, res) => {
+    //Grabs id from body request
     const userId = req.params.id;
 
     User.findOne({where: {id: userId}}).then(user => {
-        let bob =  user.getUploadedImages().then(uploads => {
-            uploads.forEach(image => {console.log(image.dataValues.url)})
+        const pics = []
+        const bob =  user.getUploadedImages().then(uploads => { 
+            // Grabs a user's uploads from DB and pushes each one into our pics array

```

Comment:
doesn't look like this variable is being used and if it is, it should be called something like userUploadedImages or something.  I'll approve because it's not affecting functionality, but it'd be good to clean up later. I think 'bob' was used in another PR too, but I just overlooked it there.